docs/development.md

```diff
@@ -51,6 +56,13 @@ applied to categorize the issue appropriately.
 Issues being worked on are moved to the **In Progress** column and assigned to
 the person working on the issue.
 
+### Versions
+
+Push repositories use a {major}.{minor}.{patch} version scheme, with
+patch tags released to address emergency issues in a deployed release.
+
```

Comment:
Do we want to specify that minors add features, majors break backs or is that horse fairly well beaten?